src/generator/models.ts

```diff
@@ -404,7 +404,11 @@ function generateStruct(lang: Language, props?: Property[]): StructDef {
     imports.add('fmt');
   }
   if (lang.responseType) {
-    imports.add("net/http");
+    imports.add('net/http');
+  }
+  if (lang.needsTimeAndContext) {
```

Comment:
I wonder if this would be better named `lang.isLRO` (or something similar)?  A bit more specific to the type.

Reply:
Yes I like that more, I was worried about having to add in other imports in the future, will update the name.

Reply:
Done